scss/modules/form.scss

```diff
@@ -40,7 +40,7 @@ input[type="text"] {
 input
 .input-description {
     margin-top: -5px;
-    font-size: 10px;
+    font-size: $smallFontSize;
```

Comment:
i have discussed this with bernd and he said that it should be 12px instead of 10px. this was not used in sulu before this and custom-url feature.